app/src/main/res/layout/fragment_my_wallets_name.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:paddingHorizontal="16dp"
+    android:paddingVertical="24dp"
+    >
+  <TextView
+      android:id="@+id/title"
+      style="@style/MyWallets.Title.Dark"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="@string/mywallet_choose_name_header"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      />
+  <ProgressBar
+      android:id="@+id/loading"
+      android:layout_width="44dp"
+      android:layout_height="44dp"
+      android:layout_marginVertical="48dp"
+      android:indeterminateDrawable="@drawable/gradient_progress"
+      android:visibility="visible"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent"
+      />
+  <com.asfoundation.wallet.ui.common.WalletTextFieldView
+      android:id="@+id/name_input"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="16dp"
+      android:maxHeight="170dp"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/title"
+      app:textFieldColor="@color/white"
+      app:textFieldHint="@string/mywallet_choose_name_field"
+      app:textFieldType="outlined"
+      />
+  <com.asfoundation.wallet.ui.common.WalletButtonView
+      android:id="@+id/save"
+      android:layout_width="wrap_content"
+      android:layout_height="@dimen/large_button_height"
```

Comment:
Looks like this should be small_button_height

Reply:
Used the same as in the Terms and Conditions bottom sheet